packages/unidecode/unidecode.0.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+
+synopsis: "Convert unicode strings into its ASCII representation"
+
+authors: [ "Daniel de Rauglaudre" ]
+
+maintainer: "Julien Sagot <julien.sagot@geneanet.org>"
+
+license: "GNU GPL"
+
+homepage: "https://github.com/geneweb/unidecode"
+
+bug-reports: "https://github.com/geneweb/unidecode/issues"
+
+dev-repo: "git+https://github.com/geneweb/unidecode.git"
+
+depends: [
+  "dune" { "1.10" }
```

Comment:
```suggestion
  "dune" { >= "1.10" }
```

If you need the specific version, you still need the `=` sign

Reply:
Sorry about that. Probably too early in the morning...